app/models/katello/repository.rb

```diff
@@ -82,9 +80,9 @@ class Repository < Katello::Model
       :attributes => :url, :nil_allowed => proc { |o| o.custom? }, :field_name => :url,
       :if => proc { |o| o.in_default_view? }
     validates :content_type, :inclusion => {
-      :in => TYPES,
+      :in => ->(_) { Katello::RepositoryTypeManager.repository_types.keys },
       :allow_blank => false,
-      :message => (_("must be one of the following: %s") % TYPES.join(', '))
+      :message => ->(_, _) { _("must be one of the following: %s") % Katello::RepositoryTypeManager.repository_types.keys.join(', ') }
```

Comment:
It means the lambda accepts 2 params neither of which we care. If I instead mark variables there, rubcop will give me an error saying "unused variables"